CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [10.1.0](https://github.com/theforeman/puppet-pulpcore/tree/10.1.0) (2024-05-16)
+
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/10.0.0...10.1.0)
+
+**Implemented enhancements:**
+
+- Allow puppet/redis 11.x and puppet/systemd 7.x [\#339](https://github.com/theforeman/puppet-pulpcore/pull/339) ([evgeni](https://github.com/evgeni))
+- use `migrate --check` not `migrate --plan |grep` check for migrations [\#338](https://github.com/theforeman/puppet-pulpcore/pull/338) ([evgeni](https://github.com/evgeni))
+- Fixes [\#37308](https://projects.theforeman.org/issues/37308) - set REMOTE\_USER properly for pulpcore registry [\#337](https://github.com/theforeman/puppet-pulpcore/pull/337) ([ianballou](https://github.com/ianballou))
```

Comment:
This sounds like a bugfix? But eck either way

Reply:
I thought the same, but @ehelms marked it as enhancement. I'll update it to bugfix.